src/analyzing/AST_to_IL.ml

```diff
@@ -1133,10 +1137,10 @@ and stmt_expr_with_pre_stmts env st =
   with_pre_stmts env (fun env -> stmt_expr env st)
 
 (* alt: could use H.cond_to_expr and reuse expr_with_pre_stmts *)
-and cond_with_pre_stmts env ?void cond =
+and cond_with_pre_stmts env cond =
```

Comment:
I think it makes sense to remove `?void` from here, but is this a clean up or a change needed for implicit-return ?

Reply:
It's a clean up that I found when I was trying to make sense of the use of `?void` in implicit returns. Not directly related.